### PR TITLE
[14.0][ADD] purchase_vendor_bill_breakdown

### PR DESCRIPTION
**Pull request by geomer198**
This module offers a solution for the following scenario:

* We purchase a **Toolbox** product from a supplier
* Supplier delivers us the Toolbox but issues an invoice with **1x Box**, **2x Wrench**, **5x Screwdriver** and **1x Hammer** products in line

This module allows to replace a product in PO Line with a predefined set of components in the vendor bill line and also allows to calculate vendors product price by product components price.

### CODE REVIEW

File: purchase_vendor_bill_breakdown/readme/USAGE.rst [hunk 1]
```diff
@@ -0,0 +1,9 @@
+* Enable **Use Product Components** option in the **Contact** section of the  **Sales & Purchases** tab.
+* Enable **Use Product Components** option in the **Products** section of the **Purchase** tab.
+* In the **Purchase** tab of the **Product**  there will be a button added for the seller.
+* Click the button and add components for the product.
```
Comment (francesco-ooops) [inline purchase_vendor_bill_breakdown/readme/USAGE.rst]: * In vendor, enable **Use Product Components** option in the **Sales & Purchases** tab.
* In product, enable **Use Product Components** option in the **Purchase** tab.
* If both product and vendor have the flag activated, in the **Purchase** tab of the **Product**, table **Vendor** there will be a button at the end of the supplierinfo line.
* Click the button and add components and quantities to display in vendor bill for the product > save
* Supplierinfo price is now readonly and calculated as sum of the first applicable supplierinfo price * their quantities.

File: purchase_vendor_bill_breakdown/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -0,0 +1,6 @@
+This module offers a solution for the following scenario:
+
+* We purchase a **Toolbox** product from a supplier
+* Supplier delivers us the Toolbox but issues an invoice with **Box**, **Wrench**, **Screwdriver** and **Hummer** products in line
```
Comment (francesco-ooops) [inline purchase_vendor_bill_breakdown/readme/DESCRIPTION.rst]: * Supplier delivers us the Toolbox but issues an invoice with **1x Box**, **2x Wrench**, **5x Screwdriver** and **1x Hammer** products in line

File: purchase_vendor_bill_breakdown/readme/USAGE.rst [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+* Create a new RQF from this Vendor
+* Hint: you can disable/enable components in the **Other information** tab of the RFQ
```
Comment (francesco-ooops) [inline purchase_vendor_bill_breakdown/readme/USAGE.rst]: remove this line

File: purchase_vendor_bill_breakdown/readme/USAGE.rst [hunk 4]
```diff
@@ -0,0 +1,9 @@
+* Enable **Use Product Components** option in the **Contact** section of the  **Sales & Purchases** tab.
+* Enable **Use Product Components** option in the **Products** section of the **Purchase** tab.
+* In the **Purchase** tab of the **Product**  there will be a button added for the seller.
+* Click the button and add components for the product.
+* Create a new RQF from this Vendor
```
Comment (francesco-ooops) [inline purchase_vendor_bill_breakdown/readme/USAGE.rst]: * Create a new RQF from this Vendor, add product with breakdown components

File: purchase_vendor_bill_breakdown/readme/USAGE.rst [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+* Hint: you can disable/enable components in the **Other information** tab of the RFQ
+* Confirm the RFQ. Not there will be a button for the component wizard in the PO Lines
```
Comment (francesco-ooops) [inline purchase_vendor_bill_breakdown/readme/USAGE.rst]: * Confirm the RFQ. Note there will be a button for the product component breakdown in the PO Lines, tracking qty received and billed

File: purchase_vendor_bill_breakdown/readme/USAGE.rst [hunk 6]
```diff
@@ -0,0 +1,9 @@
...
+* You can manually adjust the **Total quantity** of each component
```
Comment (francesco-ooops) [inline purchase_vendor_bill_breakdown/readme/USAGE.rst]: remove this line

File: purchase_vendor_bill_breakdown/readme/USAGE.rst [hunk 7]
```diff
@@ -0,0 +1,9 @@
...
+* When a vendor bill is created components are be used instead of the product in the PO Line
```
Comment (francesco-ooops) [inline purchase_vendor_bill_breakdown/readme/USAGE.rst]: * When a vendor bill is created, only received components are billed and not the product in the PO Line

**Review by francesco-ooops — CHANGES_REQUESTED** [review]

File: purchase_vendor_bill_breakdown/readme/DESCRIPTION.rst [hunk 8]
```diff
@@ -0,0 +1,6 @@
...
+
+This module allows to replace a product in PO Line with a predefined set of components in the vendor bill line and also allows to calculate vendors product price by product components price.
```
Comment (francesco-ooops) [inline purchase_vendor_bill_breakdown/readme/DESCRIPTION.rst:6]: Please note: this module is not compatible with stock. To use these features with stock components, you need to install module purchase_stock_vendor_bill_breakdown .

**Review by ivs-cetmix — APPROVED** [review]
So far so good. Please squash the commits and we are ready to go

**Pull request closed without merging**